.github/workflows/blackbox-main.yml

```diff
@@ -46,12 +46,8 @@ jobs:
       - name: Install Oracle client
         if: matrix.vendor == 'oracle'
         run: |
-          sudo apt update -y && sudo apt install -y alien libaio1 && \
-          wget https://download.oracle.com/otn_software/linux/instantclient/214000/$ORACLE_DL && \
-          sudo alien -i $ORACLE_DL && \
-          pnpm -w -D add oracledb@5
-        env:
-          ORACLE_DL: oracle-instantclient-basic-21.4.0.0.0-1.el8.x86_64.rpm
+          pnpm -w -D add oracledb@6.0.2
```

Comment:
I think the whole "Install Oracle client" block can be removed, as the dependency will exist in the `api`s package.json and is therefore installed in the regular install step